deploy/docker/Dockerfile

```diff
@@ -1,15 +1,25 @@
 FROM gcc:10 as builder
 
-COPY . /nanomq
-COPY deploy/docker/docker-entrypoint.sh /usr/bin/docker-entrypoint.sh
+COPY . /tmp/nanomq
 
 RUN apt update && apt install -y cmake
 
-WORKDIR /nanomq/build
+WORKDIR /tmp/nanomq/build
 
-RUN cmake .. && make install
+RUN cmake .. && make
 
-RUN ln -s /nanomq/build/nanomq/nanomq /usr/bin/nanomq
+FROM ubuntu:22.04
```

Comment:
This image is designed to contain the original build environment and I'm not sure if it can be use multi-from

Reply:
I've tested and it did work;